app/assets/stylesheets/trips.css.scss

```diff
@@ -1,8 +1,16 @@
 #map{
+  border-radius: 5px 5px 5px 5px;
```

Comment:
Properties should be ordered border, border-radius<br>Shorthand form for property `border-radius` should be written more concisely as `5px` instead of `5px 5px 5px 5px`